org.armagetronad.ArmagetronAdvanced.json

```diff
@@ -22,32 +19,31 @@
         "*.la", "*.a"
     ],
     "modules": [
-        "shared-modules/glu/glu-9.0.0.json",
+        "shared-modules/glu/glu-9.json",
         "shared-modules/SDL/SDL-1.2.15.json",
         "shared-modules/SDL/SDL_image-1.2.12.json",
         {
             "name": "armagetronad",
             "rm-configure": true,
             "config-opts": [
-                "--disable-games"
+                "--disable-games",
+                "--disable-uninstall",
+                "progname=org.armagetronad.ArmagetronAdvanced",
+                "CXXFLAGS=-DUSER_DATA_DIR='\"$${XDG_DATA_HOME}/game\"' -DUSER_CONFIG_DIR='\"$${XDG_CONFIG_HOME}/game\"' -DAUTORESOURCE_DIR='\"$${XDG_CACHE_HOME}/resource\"'"
```

Comment:
This is quite nonstandard. Apps usually use their name as a directory inside XDG DATA/CONFIG/CACHE directories.

So, for example, Armagetron could use:

`${XDG_DATA_HOME}/armagetronad`, `${XDG_CONFIG_HOME}/armagetronad` and `${XDG_CACHE_HOME}/armagetronad`

Reply:
Ach, of course. I hadn't realized those were the environment variables one is meant to use instead of ~/.share, ~/.cache and ~/.config when not in a sandbox. We.... already use that on 0.4, duh. The only choice here then is to make it forward compatible so on 0.4, the special defines can just go away.